test/kitchen_sink/CMakeLists.txt

```diff
@@ -142,4 +142,19 @@ if (TARGET pico_cyw43_arch)
     target_include_directories(kitchen_sink_lwip_background PRIVATE
             ${CMAKE_CURRENT_LIST_DIR})
 
+    if (PICO_MBEDTLS_PATH)
+        target_link_libraries(kitchen_sink_lwip_poll
+            pico_lwip_mbedtls
+            pico_mbedtls)
+        target_compile_definitions(kitchen_sink_lwip_poll INTERFACE
+            LIB_PICO_MBEDTLS=1)
+        target_link_libraries(kitchen_sink_lwip_background
+            pico_lwip_mbedtls
```

Comment:
what is `pico_lwip_mbedtls`? i don't see it created in this PR

Reply:
It's the mbedtls lwip "app" that we use - the alternate socket that gives us tls support. You actually added the source for this when adding lwip. See ./src/rp2_common/pico_lwip/CMakeLists.txt

Reply:
oh duh... sorry